fox_news_scraper/test_extract_fn.py

```diff
@@ -0,0 +1,31 @@
+# pylint: skip-file
+
+from unittest.mock import patch, MagicMock
+import feedparser
```

Comment:
Not major important, just personal preference, but I do ones you don't need to pip install (unittest..) then space, ones you need to pip install (feedparser) then space, then all your own functions (extract stuff)

Reply:
Good point, I'll make that change now!